DDEV v1.24.0 release announcement

rfay opened the pull request:
## The Issue

DDEV v1.24.0 release announcement

See rendered version at https://20241127-release-v1-24-0.ddev-com-front-end.pages.dev/blog/release-v1240-php84/

Related draft that may need parallel updates, https://github.com/ddev/ddev/issues/6773

## TODO
- [ ] Consider installation instructions

tyler36 commented on src/content/blog/release-v1.24.0.md:
```diff
@@ -0,0 +1,48 @@
+* The `drupal` project type is no longer a generic project type, but instead is an alias to the latest stable Drupal (`drupal11` right now). A number of folks gave feedback that it caused the startup time to be longer and the process more complex. (`ddev config --update` is no longer a part of the Drupal quickstart)
+* In v1.23.5 we started preferring vendor/bin/composer to the specified `composer_version` but users thought that was a mistake, so this was reverted
+* The nginx-proxy router has been removed.
+* The `ddev service enable` and `ddev service disable` commands were removed, as they have long been superceded by the use of `ddev add-on get` and `ddev add-on remove`
```
```suggestion
* The `ddev service enable` and `ddev service disable` commands were deprecated, as they have long been superceded by the use of `ddev add-on get` and `ddev add-on remove`
```

rpkoller commented on src/content/blog/release-v1.24.0.md:
```diff
@@ -0,0 +1,48 @@
+
+* Added dynamic timeout handling for scripts on start based on setting of `default_container_timeout`.
+* `ddev describe` changes how exposed ports are displayed for better clarity (thanks @ hanoii)
+* Only a limited set of locales is now installed in ddev-webserver by default. If you need more, use `webimage_extra_packages: [locales-all]`
```
```suggestion
* Only a limited set of locales is now installed in `ddev-webserver` by default. If you need more, use `webimage_extra_packages: [locales-all]`
```
line 27 uses single quotes for the image as well

rpkoller commented on src/content/blog/release-v1.24.0.md:
```diff
@@ -0,0 +1,48 @@
+
+* The `drupal` project type is no longer a generic project type, but instead is an alias to the latest stable Drupal (`drupal11` right now). A number of folks gave feedback that it caused the startup time to be longer and the process more complex. (`ddev config --update` is no longer a part of the Drupal quickstart)
+* In v1.23.5 we started preferring vendor/bin/composer to the specified `composer_version` but users thought that was a mistake, so this was reverted
+* The nginx-proxy router has been removed.
```
```suggestion
* The `nginx-proxy` router image has been removed.
```
also wrapped the image in single quotes and also appended image to make it more explicit here.

rfay replied:
Actually, the image hasn't been removed, the option of using it has been removed.

```suggestion
* The `nginx-proxy` router has been removed and is no longer an option.
```

rfay commented on src/content/blog/release-v1.24.0.md:
```diff
@@ -0,0 +1,48 @@
+summary: New features and changes in DDEV v1.24.0
+author: Randy Fay
+featureImage:
+  src: /img/blog/2024/12/php-logo.svg
```
Thanks, pushed.

ptmkenny commented on src/content/blog/release-v1.24.0.md:
```diff
@@ -0,0 +1,48 @@
+
+* Added dynamic timeout handling for scripts on start based on setting of `default_container_timeout`.
+* `ddev describe` changes how exposed ports are displayed for better clarity (thanks [@hanoii](https://github.com/hanoii))
+* Only a limited set of locales is now installed in `ddev-webserver` by default. If you need more, use `webimage_extra_packages: [locales-all]`
```
To reduce the image size of `ddev-webserver`, only a few locales are installed by default. To install the full range of locales, set `webimage_extra_packages: [locales-all]` in `.ddev/config.yaml`.

ptmkenny commented on src/content/blog/release-v1.24.0.md:
```diff
@@ -0,0 +1,48 @@
+
+**Reversions and Removals**
+
+* The `drupal` project type is no longer a generic project type, but instead is an alias to the latest stable Drupal (`drupal11` right now). A number of folks gave feedback that it caused the startup time to be longer and the process more complex. (`ddev config --update` is no longer a part of the Drupal quickstart)
```
The `drupal` project type is now an alias to the latest stable major version of Drupal (currently, `drupal11`). A number of folks noted that using `drupal` as a generic project type (the old way `ddev` handled this) caused the startup time to be longer and made the process more complex. (`ddev config --update` is no longer part of the Drupal quickstart.)

ptmkenny reviewed (commented):
Suggested slightly more detailed explanations for two changes related to Drupal.

bmartinez287 commented:
<img width="569" alt="Screenshot 2024-12-02 at 5 07 59 PM" src="https://github.com/user-attachments/assets/66efd52a-4855-4ec9-b2b1-bb2b35c9b248">
That PHP pic on top could be better. I like the AI ones a bit more. Plus it being an SVG complicates things since it does not have a background and we rarely used those.

bmartinez287 commented on src/content/blog/release-v1.24.0.md:
```diff
@@ -0,0 +1,48 @@
+* The `ddev service enable` and `ddev service disable` commands were deprecated, as they have long been superceded by the use of `ddev add-on get` and `ddev add-on remove`
+* Removed support for Python and Django4 projects. After more than a year, these did not gain traction or a community, so sadly we had to leave them behind.
+
+Again, thanks so much to all of you for using, promoting, contributing, and supporting DDEV!
```
As the DDEV team, we appreciate your support and contributions to DDEV.  Let's work together to improve productivity and performance in 2025!
```suggestion
Again, thanks so much to all of you for using, promoting, contributing, and supporting DDEV!
```

rfay replied:
I'm desperately trying to figure out what your suggestion was.

bmartinez287 replied:
ups, I just fixed that. I had left it on top.

rfay replied:
Sorry, that seems awkward to me. It's longer and less direct. What is it you're trying to accomplish here?

bmartinez287 replied:
Primarily, to reference the DDEV team and skip "so much".

rfay commented:
@bmartinez287 I changed it to use the other PHP logo, do you like it better?

bmartinez287 commented:
Yeah!
![image](https://github.com/user-attachments/assets/764d8fd5-ddbd-4635-9e09-28cbb2948fa9)
That's the timeless one.

rfay commented:
@bmartinez287 I went wild and removed twitter/x and added Bluesky. Should have been separate PR but wanted it to be here for this. Could you see if I made any mistakes?

bmartinez287 commented:
<img width="247" alt="Screenshot 2024-12-02 at 7 04 21 PM" src="https://github.com/user-attachments/assets/735474ee-2479-4ea4-a602-f477300ec23e">
We don't have the icon for bluesky

rfay commented:
If you know how to add one, could you push it in here?

rfay commented:
It was also missing from any place that wanted it in 
* https://github.com/ddev/ddev/pull/6799
